CryptoPy/cryptoScraper.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+from requests.exceptions import ConnectionError, Timeout, TooManyRedirects
+
```

Comment:
Create a variable `Currency = 'BTC'` and replace `'BTC` with `Currency`at line 32